### PR TITLE
More e2e tests for Firefly projects

### PR DESCRIPTION
**Pull request by moritzraho**
## Description
relates to https://github.com/adobe/aio-cli-plugin-app/pull/376

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: e2e/e2e.js [hunk 1]
```diff
@@ -95,7 +96,7 @@ describe('organizations', () => {
 })
 
 describe('create, edit, get', () => {
-  test('test createFireflyProject API', async () => {
+  test('test createFireflyProject API with trailing spaces in title and description', async () => {
```
Comment (shazron) [inline e2e/e2e.js:99]: the title and description do not have trailing spaces here?

File: e2e/e2e.js [hunk 2]
```diff
@@ -140,6 +141,24 @@ describe('create, edit, get', () => {
...
   })
 
+  test('test editProject API (firefly project type and with trailing spaces in title and description)', async () => {
+    expect(orgId).toBeDefined()
+    expect(fireflyProjectId).toBeDefined()
+
+    const res = await sdkClient.editProject(
+      orgId,
+      fireflyProjectId,
+      { name: fireflyProjectName, description: modifiedProjectDescription, title: 'modified project title', type: 'jaeger' }
```
Comment (shazron) [inline e2e/e2e.js:151]: the title and description do not have trailing spaces here?

File: e2e/e2e.js [hunk 3]
```diff
@@ -207,7 +227,37 @@ describe('create, edit, get', () => {
     console.log('Workspace created with Id: ' + workspaceId)
   })
 
-  test('test getWorkspacesForProject API', async () => {
+  test('test createWorkspace API for firefly project type (with trailing spaces in title and description)', async () => {
+    expect(orgId).toBeDefined()
+    expect(fireflyProjectId).toBeDefined()
+    const res = await sdkClient.createWorkspace(orgId, fireflyProjectId, { name: fireflyWorkspaceName, title: 'workspace title', description: workspaceDescription })
```
Comment (shazron) [inline e2e/e2e.js:233]: the title and description do not have trailing spaces here?

**Review by shazron — CHANGES_REQUESTED** [review]
Some questions regarding the label for the tests not matching the actual test.

**Comment by moritzraho** [comment]
oops forgot to remove the label as I moved all whitespace tests to the end of the file. Thanks @shazron, pushed a fix commit

**Review by shazron — APPROVED** [review]

**Pull request merged**